src/index.js

```diff
@@ -76,6 +77,7 @@ const resetIndexes = (arr) => arr.forEach((item, idx) => { item.index = idx + 1;
 const deleteTask = (task) => {
   const item = storedToDos[task];
   storedToDos = storedToDos.filter((todo) => todo !== item);
+  console.log(storedToDos);
```

Comment:
- [ ] [OPTIONAL] please remove the 2 `consle.log` it will make your application work better and your codebase will look a lot cleaner.

Reply:
Thanks for your review :+1: 
I'd fix all the issues immediately